### ISSUE TITLE
 Failed to resolve entry

### ISSUE THREAD

**Issue by newtondotcom** [joshnuss/svelte-stripe#93]
### Describe the bug

```Error: Failed to resolve entry for package "svelte-stripe". The package may have incorrect main/module/exports specified in its package.json.```

### Reproduction

my depencies
```
	"devDependencies": {
		"@stripe/stripe-js": "^2.1.11",
		"@sveltejs/adapter-auto": "^2.0.0",
		"@sveltejs/adapter-node": "^1.3.1",
		"@sveltejs/kit": "^1.20.4",
		"autoprefixer": "^10.4.16",
		"postcss": "^8.4.31",
		"stripe": "^14.5.0",
		"svelte": "^4.0.5",
		"svelte-check": "^3.4.3",
		"svelte-stripe": "^1.1.1",
		"tailwindcss": "^3.3.5",
		"tslib": "^2.4.1",
		"typescript": "^5.0.0",
		"vite": "^4.4.2"
	},
```
the command i used
```
pnpm install -D stripe svelte-stripe
```

### Severity

blocking all usage

### Additional Information

I ve tried to add ```noExternal: ['svelte-stripe-js'],``` to ```svelte.config.js``` without success

**Comment by MatthewNobes** [comment]
I had the same problem. I did a temporary fix by downgrading svelte-stripe to version 0.0.24, and it works. Not ideal but if you need someway of testing this package quick it can work till a fix is done.

**Comment by newtondotcom** [comment]
@MatthewNobes  it works for now thanks !

**Comment by joshnuss** [comment]
@newtondotcom  Sorry about that! and thanks for reporting.

The issue was that I added `semantic-release` to auto-deploy this package. But it was configured incorrectly. It deployed the root of the project, when it should have deployed only the `package` subfolder. ie `pnpm package && cd package && semantic-release`

The issue is now resolved in bbee6e06b6ccbb7c643a4d28880375961201e9bb
It's deployed in v1.1.2.
